src/components/presentational/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -224,24 +208,25 @@ export default function TimeSeriesChart(props: TimeSeriesChartProps) {
                         ticks={xAxisTicks}
                         allowDataOverflow
                         interval={0}
+                        {...props.options?.xAxisOptions}
                     />
                     {props.children}
                     {props.chartHasData &&
                         <>
                             {props.chartType === "Line" &&
                                 <>
                                     {createLineChartDefs(layoutContext, gradientKey, props.series, props.options, keys, dataToDisplay!)}
-                                    {(props.options as MultiSeriesLineChartOptions)?.thresholds?.filter(t => t.referenceLineColor)?.map(threshold =>
-                                        <ReferenceLine y={threshold.value} stroke={resolveColor(layoutContext.colorScheme, threshold.referenceLineColor)} />
+                                    {(props.options as MultiSeriesLineChartOptions)?.thresholds?.filter(t => t.referenceLineColor)?.map((threshold, index) =>
+                                        <ReferenceLine key={index} y={threshold.value} stroke={resolveColor(layoutContext.colorScheme, threshold.referenceLineColor)} />
```

Comment:
add a prefix to the key (example key={`refLine_${index}`}

Reply:
I've updated them to be more specific keys.